mlflow/pyfunc/cli.py

```diff
@@ -1,13 +1,54 @@
 from __future__ import absolute_import
 
 import sys
+import os
+import subprocess
 
 import click
 import pandas
 
-from mlflow.pyfunc import load_pyfunc, scoring_server
+from mlflow.pyfunc import load_pyfunc, scoring_server, load_model_env
 from mlflow.tracking import _get_model_log_dir
 from mlflow.utils import cli_args
+from mlflow.projects import _maybe_create_conda_env, _get_conda_env_name
+import six
+
+
+def _rerun_in_conda(conda_env_path):
+    """ Rerun CLI command inside a to-be-created conda environment."""
+    conda_env_name = _get_conda_env_name(conda_env_path)
```

Comment:
Could we change `_maybe_create_conda_env` to return the name of the created conda_env? That way we only have to invoke that one.

Reply:
good idea.